.github/workflows/ci.yml

```diff
@@ -7,16 +7,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: ['10.1', 10.x, '12.1', 12.x, '14.1', 14.x]
+        script:
+          - 'test'
```

Comment:
I believe we need to also manually add `lint` here if we're going to run tests using `--ignore-scripts`, I do like that change since that's going to highlight much better in the GH actions UI when there are **lint** errors vs **test** errors:

```suggestion
          - 'test'
          - 'lint'
```

Reply:
I noticed there's also:

```
        include:
          - node-version: '14.x'
            script: 'posttest'
            platform: { 'os': 'ubuntu-latest', 'shell': 'bash' }
```

but still, I think having it as a separate script will yield a better UI when parsing through the GH actions results and we can also limit to run in a single target (similar to how we did with coverage) if wanted

Reply:
The linter runs in `posttest` - the intention here is that no matter what `npm test` does, CI will currently run it.

Reply:
Specifically, using `include` is quite intentional - the linter should only ever run once, and currently, it's running on every CI job.

eslint itself requires node 10.12, so the linter will fail on the .0 or .1 tests anyways.

Reply:
yeah sorry my messages were confusing I get all that, what I mean by "GH actions UI" is having the lint script run in a way that it appears as a separated item in the UI, similar to "Run Coverage Report" and "Validate licenses" here:

<img width="465" alt="Screen Shot 2021-02-12 at 10 36 12 AM" src="https://user-images.githubusercontent.com/220900/107788377-548e7200-6d1e-11eb-868f-37855bdbf403.png">